Sming/SmingCore/Network/MqttClient.cpp

```diff
@@ -71,15 +121,21 @@ bool MqttClient::connect(const String& clientName, const String& username, const
 
 	debug_d("MQTT start connection");
 	if (clientName.length() > 0)
-		 mqtt_set_clientid(&broker, clientName.c_str());
+	{
+		mqtt_set_clientid(&broker, clientName.c_str());
+	}
 
 	if (username.length() > 0)
+	{
 		mqtt_init_auth(&broker, username.c_str(), password.c_str());
+	}
 
-	if(server.length() > 0 ) {
+	if (server.length() > 0 ) 
```

Comment:
For the `if` statements - please put the opening curly brace on the same line after the if condition. Separate the opening curly brace and the condition with one space. Apply that for all ifs in this class.